pom.xml

```diff
@@ -25,17 +25,92 @@
 		<dcsa.artifacttype>-SNAPSHOT</dcsa.artifacttype>
 	</properties>
 
+
 	<dependencies>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-data-r2dbc</artifactId>
```

Comment:
`dcsa_core` and  `dcsa_event_core` already include all these dependencies, so you do not need to add them here.